src/SignUpIn/DelivererGroup/DelivererGroupSignUpController.js

```diff
@@ -51,6 +53,50 @@ class DelivererGroupSignUpController extends Component {
         });
     }
 
+    submitRegistration() {
+        // Handle via ajax submitting the user data, upon
+        // success return this.nextStop(). If it fails,
+        // show the user the error but don't advance
+        
+        firebase.auth().createUserWithEmailAndPassword(fieldValues.email, fieldValues.password)
+            .then(user => {
+                let postData = {
+                    accountType: 'deliverer_group',
+                    name: fieldValues.organizationName,
+                    numVolunteers: fieldValues.numVolunteers,
+                    address: {
+                        street1: fieldValues.address1,
+                        street2: fieldValues.address2,
+                        city: fieldValues.city,
+                        state: fieldValues.state,
+                        zip: fieldValues.zip
+                    },
+                    email: fieldValues.email,
+                    coordinator: {
+                        name: fieldValues.contactName,
+                        email: fieldValues.contactEmail,
+                        phone: fieldValues.contactNumber,
+                        position: fieldValues.contactPosition
+                    },
+                    school: 'RheaQY1WxJT03sTPQICFZ4STpfm1',
+                    isActivated: true,
+                    isVerified: true,
+                    notification: {
+                        type: NotificationType,
+                        content: '-L5QoXeC_UrL5tRRED3e'
+                    }
+                };
+                let updates = {};
+                updates['/accounts/' + user.uid] = postData;
+                return firebase.database().ref().update(updates);
```

Comment:
You should also be add the uid of newly created agencies to the lists under the `umbrella` account. In this case, you should add it to the `umbrella`'s `delivererGroups` field.

To do test this, please create a new `umbrella` account to test with since I'm using the current one to test for other things. Please:

1. Manually create a new `umbrella` account, follow `RheaQY1WxJT03sTPQICFZ4STpfm1`. (You can leave the lists empty since you're going to add to it when you're testing.)
2. I suggest you add the new `umbrella` uid to the Enums file (this is temporary, for when we are manually setting the `umbrella` field. Add something like:
```
exports.UmbrellaAccount = {
    TEST = '<your_new_uid>',
    // we will add the real UW's later
};
```
3. Change all the `umbrella` in this PR to the new uid, and add the code that adds the new agency uid to the umbrella's lists.